File: fizzbuzz.cs
class fizzbuzz
{
	static void Main() {
		for(int i = 1; i <= 15; i++) {
			string output = "";
			if(i % 3 == 0){
				output += "fizz";
			}
			if(i % 5 == 0){
				output += "buzz";
			}
			System.Console.WriteLine(output == "" ? i.ToString() : output);
		}
	}
}

File: fizzbuzz.rs
fn main() {
    for i in 1..(15 + 1) {
        let output = String::from(if i % 3 == 0 { "fizz" } else { "" }) + (if i % 5 == 0 { "buzz" } else { "" });
        println!("{}", if output != "" { output } else { i.to_string() });
    }
}
